src/components/PostForm/components/TextArea/TextArea.tsx

```diff
@@ -0,0 +1,37 @@
+import { TextField } from '@mui/material'
+import { ReactElement } from 'react'
+
+import { PostFieldDataValues } from '../../../../lib/queries/getPost'
+import type { PostFieldProps } from '../../FieldRenderer'
+
+export function TextArea({
+  field,
+  value,
+  error,
+  helperText,
+  required,
+  onBlur,
+  onChange
+}: PostFieldProps): ReactElement {
+  return (
+    <TextField
+      data-testid="text area"
```

Comment:
same for this, this seems like for testing purposes. I feel like we could just use findByRole and look for the name attribute.